src/tests/unit/tests/injected/components/details-dialog.test.tsx

```diff
@@ -101,5 +106,14 @@ describe('DetailsDialogTest', () => {
         (testObject as any).onLayoutDidMount = onLayoutDidMountMock;
 
         expect(testObject.render()).toMatchSnapshot();
+        const wrapper = shallow(<DetailsDialog {...props} />);
+        if (!shadowDialog) {
+            expect(
+                wrapper
+                    .find(Dialog)
+                    .props()
+                    .dialogContentProps.topButtonsProps[0].onRenderIcon(),
+            ).toMatchSnapshot('verify close button for non shadow dom');
```

Comment:
how is this not being tested by line 108? I'm a little confused...

Reply:
Using `render` or `mount` did not trigger the private function that I added (`renderCloseIcon`), so I added this to manually call the function.